### DISCUSSION TITLE
Path to exe

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#11415] [General - Components / Directives / etc]
How can i get the path to the executable of my Electron Application? Is that possible to do...

**Comment by hawkeye64** [comment]
At build time? Runtime? Need more details...

**Reply by JokerMartini** [reply]
path to exe at build time

**Reply by hawkeye64** [reply]
I was thinking you can use `afterDev` and `afterBuild` hooks, but I think those just pass in the quasar conf. What you need is the quasar/app instance. It has a `appPaths` you can use like so: `appPaths.resolve.electron('icons')`. But, I don't think that is passed in. You could make a feature request if this is suitable (meaning, you will have to look at quasar/app src first).
